lib/ui/text.dart

```diff
@@ -432,14 +432,43 @@ enum TextDecorationStyle {
   wavy
 }
 
+/// {@macro dart.ui.leadingDistribution}
+enum LeadingDistribution {
+  /// Distributes the ["leading"](https://www.w3.org/TR/CSS2/visudet.html#leading)
```

Comment:
let's link to https://en.wikipedia.org/wiki/Leading rather than the CSS spec though

Reply:
Would it be better to avoid using the term altogether then? The Wikipedia page says "the exact definition varies", and "In electronic typesetting, "leading" often instead means the distance from one baseline to the next. ".